elements/pl-file-editor/pl-file-editor.js

```diff
@@ -19,6 +19,7 @@ window.PLFileEditor = function (uuid, options) {
   this.editor.setTheme('ace/theme/chrome');
   this.editor.getSession().setUseWrapMode(true);
   this.editor.setShowPrintMargin(false);
+  this.editor.setReadOnly(!!options.readOnly);
```

Comment:
Nit: this should always be defined, so the `!!` shouldn't be necessary.
```suggestion
  this.editor.setReadOnly(options.readOnly);
```